src/nih_wayfinding/app/scripts/views/profile/new-profile/new-profile-controller.js

```diff
@@ -2,50 +2,108 @@
     'use strict';
 
     /* ngInject */
-    function NewProfileController($state, ProfileService) {
-
+    function NewProfileController($state, ProfileService, Notifications) {
         var ctl = this;
-        var usingWheelchairScooter = false;
-        var challengeLevel = 2;
-
         initialize();
 
         function initialize() {
             ctl.step = 1;
             ctl.setStep = setStep;
-            ctl.checkUsername = checkUsername;
-            ctl.usingWheelchair = usingWheelchair;
-            ctl.setChallengeLevel = setChallengeLevel;
-            ctl.willCreateLocations = willCreateLocations;
+            ctl.newUser = ProfileService.createBaseProfile();
+
             ctl.errorMsg = '';
             ctl.displayUsername = '';
 
-            ctl.yesNoOpts = [{
+            // Values pertaining to questionnaire
+            ctl.checkUsername = checkUsername;
+
+            ctl.wheelchairUseOpts = [{
                 text: 'Yes',
                 value: true
             }, {
                 text: 'No',
                 value: false
             }];
+            ctl.setUsingWheelchair = setUsingWheelchair;
 
-            ctl.newLocOpts = [{
-                text: 'Create locations',
+            ctl.wheelchairTypeOpts = [{
+                text: 'Electric',
                 value: true
             }, {
-                text: 'No thanks',
+                text: 'Manual',
                 value: false
             }];
+            ctl.wheelchairTypeDialog = false;
+            ctl.setWheelchairType = setWheelchairType;
 
             ctl.challengeOpts = [{
                 text: 'Minimal',
-                value: 1
+                value: 0.1
             }, {
                 text: 'Moderate',
-                value: 5
+                value: 0.5
             }, {
-                text: 'Challenge Me',
-                value: 10
+                text: 'Challenge me',
+                value: 1
             }];
+            ctl.setChallengeLevel = setChallengeLevel;
+
+            ctl.steepTerrainOpts = [{
```

Comment:
It would probably make sense to move all of these to a single object on a constant service, to clean up this controller a bit.